jina/serve/consensus/jina.proto

```diff
@@ -55,9 +55,11 @@ message EndpointsProto {
 
     // list of endpoints exposed by an Executor
     repeated string endpoints = 1;
-    repeated string write_endpoints = 2;
+    // TODO(niebayes): regenerate proto files.
+    repeated string read_endpoints = 2;
```

Comment:
no need to add an extra list, the difference between all the endpoints and write endpoints is already the read endpoints

Reply:
Are there any endpoints not for read nor for write? For e.g. endpoints about shutting down an executor, or about upgrade an executor?
On the other hand, some endpoints may not want to involve Raft at all, i.e. we simply forward them to the executor directly rather letting raft worker forward them.

Reply:
There are only write and read.